code/src/java/pcgen/cdom/enumeration/AssociationKey.java

```diff
@@ -126,8 +126,7 @@ public T cast(Object obj)
 		return (T) obj;
 	}
 
-	public static <OT> AssociationKey<OT> getKeyFor(Class<OT> assocClass,
-			String assocName)
+	public static <OT> AssociationKey<OT> getKeyFor(String assocName)
```

Comment:
While the parameter isn't used, this change actually alters a form of type safety in code that is used to protect from quirks in the generics system in Java